runbotics/runbotics-orchestrator-ui/src/views/process/ProcessBuildView/Modeler/extensions/custom/CustomRenderer.js

```diff
@@ -64,11 +63,25 @@ export default class CustomRenderer extends BaseRenderer {
         });
 
         // svgClasses(text).add('djs-label');
-        const translatedLabel = translate(label.translateKey)
 
-        svgAppend(text, document.createTextNode(label.title ? label.title : translatedLabel));
+        const actionId = label.actionId;
+
+        if(actionId) {
+            const translateKey = `Process.Details.Modeler.Actions.${(actionId.split('.')).reduce((prev, curr) => {
```

Comment:
another repetition

Reply:
each change in architecture and then you have to search for every usage like this

Reply:
done